RELEASE.md

```diff
@@ -1,3 +1,29 @@
+# Release 0.8.0
+
+## Major Features
+* `tensorflow_io.json`: JSON Dataset support.
+* `tensorflow_io.dicom`: Medical image DICOM format support.
+* `tensorflow_io.genome`: DNA Sequence FastQ format support added.
+* `tensorflow_io.ffmpeg`: FFmpeg now support selection of substreams.
+* `tensorflow_io.ffmpeg`: FFmpeg now support subtitle (captioning).
+* `tensorflow_io.ffmpeg`: FFmpeg now support decode video from memory.
+* `tensorflow_io.image`: BMP encoding (encode_bmp) support.
+* `tensorflow_io.kafka`: Kafka Dataset now support Kafka Schema Registry.
+* `tensorflow_io.audio`: WAV format now support 24 bit audio streams.
+* `tensorflow_io.text`: Regex capture group (`re2_full_match`) support.
+* manylinux2010 compliant on Linux.
+
+## Thanks to our Contributors
+
+This release contains contributions from many people:
+
+Aleksey Vlasenko, Amarpreet Singh, Bryan Cutler, Damien Pontifex, Duke Wang,
+Jiacheng Xu, Marcelo Lerendegui, Mark Daoust, Ouwen Huang, Suyash Kumar,
+Yong Tang, Yuan Tang, android
```

Comment:
Who's "android"?

Reply:
@terrytangyuan android (henrytansetiawan).

Reply:
We should probably used henrytansetiawan instead? I understand that this list is generated automatically but we should probably manually edit it if needed.

Reply:
@terrytangyuan Update.